src/Neo.Cryptography.BLS12_381/Fp.cs

```diff
@@ -152,11 +153,7 @@ public bool TryWrite(Span<byte> buffer)
 
         public override string ToString()
         {
-            var output = string.Empty;
-            foreach (var b in ToArray())
-                output += b.ToString("x2");
-
-            return "0x" + output;
+            return "0x" + ToArray().ToHexString();
```

Comment:
How this change is related to `MemoryExtenions.SequenceEqual`?

Reply:
> How this change is related to `MemoryExtenions.SequenceEqual`?

No. Just a performance improvement.